pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -58,49 +61,82 @@ const (
 var cleanUpRoutineInitialized = false
 var datastoreFolderIDMap = make(map[string]map[string]string)
 
-var clientLock sync.Mutex
 var cleanUpRoutineInitLock sync.Mutex
 var cleanUpDummyVMLock sync.RWMutex
 
 // VSphere is an implementation of cloud provider Interface for VSphere.
 type VSphere struct {
+	cfg      *VSphereConfig
+	hostName string
```

Comment:
what is hostName, could you all comments?

Reply:
hostName is the OS hostName that vsphere cloud provider will report.

Reply:
hostName is verbose enough name. Comment will just say something like "hostName is OS host name"

Reply:
that's fine then. I just wondered why adding this new field

Reply:
Earlier we instanceID was used as variable name. Now hostName is used as variable name.